README.md

```diff
@@ -41,13 +41,14 @@ The login flow needs at least Nextcloud 16 to work.
 ```plain
 $ nextcloud-exporter --help
 Usage of nextcloud-exporter:
-  -a, --addr string          Address to listen on for connections. (default ":9205")
-  -c, --config-file string   Path to YAML configuration file.
-      --login                Use interactive login to create app password.
-  -p, --password string      Password for connecting to Nextcloud.
-  -s, --server string        URL to Nextcloud server.
-  -t, --timeout duration     Timeout for getting server info document. (default 5s)
-  -u, --username string      Username for connecting to Nextcloud.
+  -a, --addr string            Address to listen on for connections. (default ":9205")
+  -c, --config-file string     Path to YAML configuration file.
+      --login                  Use interactive login to create app password.
+  -p, --password string        Password for connecting to Nextcloud.
+  -s, --server string          URL to Nextcloud server.
+  -t, --timeout duration       Timeout for getting server info document. (default 5s)
+  -u, --username string        Username for connecting to Nextcloud.
+  --tls-skip-verify boolean    Disables checking HTTPS certificates of Nextcloud.
```

Comment:
I usually use the actual output of the tool to create this. Noticed that I forgot to update this with the last PR:

```suggestion
      --tls-skip-verify      Skip certificate verification of Nextcloud server.
  -u, --username string      Username for connecting to Nextcloud.
  -V, --version              Show version information and exit.
```